app/components/Header/DropdownContents/ProfileDropdown.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import styles from '../Header.css';
+
+const ProfileDropdown = () => (
+  <div className={styles.profileDropdownEl}>
+    <div className={styles.dropdownSection} data-first-dropdown-section>
+      <div>Ola Nordmann</div>
+      <div>Innstillinger</div>
+      <div>Møteinnkallinger</div>
```

Comment:
I'm guessing this is WIP? Or at least the Ola Nordmann Part

Reply:
Yes. I needed to make a bunch of dummy components in order to test out the core functionality ;)